lib/led/ledcontrol.js

```diff
@@ -59,9 +59,8 @@ var priv = new Map(),
  *                                    Only for use with the HT16K33 controller.
  */
 function LedControl(opts) {
-
   Board.Component.call(
-    this, opts = Board.Options(opts)
+    this, this.opts = opts = Board.Options(opts)
```

Comment:
Don't assign `opts` to `this.opts`. Some classes have this and it's a mistake that needs to be rectified